config/management/src/smoke_test.rs

```diff
@@ -75,21 +89,19 @@ fn smoke_test() {
         validator_network.identity = Identity::from_storage(
             libra_global_constants::VALIDATOR_NETWORK_KEY.into(),
             libra_global_constants::OPERATOR_ACCOUNT.into(),
-            secure_backend(helper.path(), &swarm_path, &ns, "validator"),
+            self.secure_backend(&ns, "validator"),
         );
 
         let fullnode_network = &mut config.full_node_networks[0];
         let fullnode_network_address = fullnode_network.listen_address.clone();
         fullnode_network.identity = Identity::from_storage(
             libra_global_constants::FULLNODE_NETWORK_KEY.into(),
             libra_global_constants::OPERATOR_ACCOUNT.into(),
-            secure_backend(helper.path(), &swarm_path, &ns, "full_node"),
+            self.secure_backend(&ns, "full_node"),
         );
 
-        configs.push(config);
-
-        helper.operator_key(&ns, &ns_shared).unwrap();
-        helper
+        self.storage_helper.operator_key(&ns, &ns_shared).unwrap();
```

Comment:
I think this might be a bug in the existing code. There shouldn't be any need to call "self.storage_helper.operator_key(..)" twice (see line 81 above)? You should be able to remove this (e.g., tested this locally on master and it seemed to work).